src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureScheduler.cpp

```diff
@@ -33,14 +33,47 @@
 // --------------------------------------------------------------------------
 
 #include <OpenMS/ANALYSIS/OPENSWATH/MRMFeatureScheduler.h>
+#include <OpenMS/ANALYSIS/OPENSWATH/MRMFeatureSelector.h>
+#include <OpenMS/KERNEL/FeatureMap.h>
 
 namespace OpenMS
 {
   MRMFeatureScheduler::MRMFeatureScheduler() {}
 
   MRMFeatureScheduler::~MRMFeatureScheduler() {}
 
-  // TODO I guess we are supposed to use setParameters()
-  void MRMFeatureScheduler::schedule_MRMFeatures_qmip() {}
-  void MRMFeatureScheduler::schedule_MRMFeatures_score() {}
+  FeatureMap MRMFeatureScheduler::schedule_MRMFeatures(MRMFeatureSelector& feature_selector, FeatureMap& features) {
+    int nn_thresholds[] = {2, 4, 6, 4};
+    bool locality_weights[] = {false, true, true, true};
+    bool select_transition_groups[] = {true, true, true, true};
+    int segment_window_lengths[] = {12, 24, 48, -1};
+    int segment_step_lengths[] = {2, 6, 12, -1};
+    bool select_highest_counts[] = {false, false, false, false};
+    String variable_types[] = {"continous", "continous", "continous", "continous"};
+    double optimal_thresholds[] = {0.5, 0.5, 0.5, 0.5};
+    FeatureMap optimal_features = features;
+    for (size_t i = 0; i < 2; ++i) {
```

Comment:
Instead of using `2`, please explain its purpose by defining a constant like:
`const size_t identifier = 2;`